index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/bf723a76a7.js" crossorigin="anonymous"></script>
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <nav>
+        <h3> Ikpro </h3>
+        <a href="#"><i class="fa-solid fa-bars"></i></a>
+    </nav>
+    <h1>Inayat</h1>
+    <h2> Software Developer</h2>
+    <p>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like
+        what you
+        see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <hr>
```

Comment:
- [ ] Don't you think wrapping this entire piece of code in a `<section>`  tag would be a good idea. It'll help with the semantic structure and help contain your headline so you can add an overall padding and margin to the section instead of adding paddings and margins to each individual element. It'll even help you align your content using `display: flex` for the section
- [ ] Also, you seem to have missed the background image that was supposed to be inside this section. Kindly make sure you follow the Figma design exactly and include the background image as can be seen below.
> Yours 

![image](https://user-images.githubusercontent.com/90556221/164559140-75991976-e705-4631-8b83-3ef193b29cd6.png)
> Figma

![image](https://user-images.githubusercontent.com/90556221/164433657-55320c6e-b64a-4181-8a0f-f2140ba8329a.png)

You can use the CSS[ `background` or `background-image`](https://css-tricks.com/almanac/properties/b/background/) property for this after you create your section and download the image needed from the Figma template (Click on it and you'll see download options in the right menu)